pkg/controller/common/reconcile_test.go

```diff
@@ -0,0 +1,220 @@
+package common
+
+import (
+	"testing"
+
+	ign3types "github.com/coreos/ignition/v2/config/v3_4/types"
+	"github.com/openshift/machine-config-operator/pkg/daemon/constants"
+	"github.com/openshift/machine-config-operator/test/helpers"
+)
+
+// TestReconcilable attempts to verify the conditions in which configs would and would not be
```

Comment:
Note to reviewer: These test cases were moved from `pkg/daemon` verbatim. I did modify their names slightly and write additional test cases to verify the forcefile behavior as well as the FIPS behavior.